static/js/directives/time-ago-auto-update.js

```diff
@@ -0,0 +1,59 @@
+var moment = require('moment');
+
+/**
+ * Directive for an auto-updating relative time ago date.
+ * Renders only text and auto-updates it. For better display should be
+ * used in other directives.
+ */
+angular.module('inboxDirectives').directive('timeAgoAutoUpdate', ['$interval', 'FormatDate', function($interval, FormatDate) {
+  'use strict';
+  'ngInject';
```

Comment:
If you use 'ngInject' you don't need to specify the array of dependencies, so you can just use:

```js
angular.module('inboxDirectives').directive('timeAgoAutoUpdate', function($interval, FormatDate) {
```

If you felt like it, also update the other directives to use 'ngInject'.

Reply:
Neat